metadata/extension-readme.md

```diff
@@ -0,0 +1,21 @@
+# ADO Extension for Defender for DevOps
+
+An extension for Azure DevOps that allows [Microsoft Defender for DevOps](https://aka.ms/DfDdocs) to extract metadata from pipelines, such as containers digest ID, container image name etc. That information will then be used within [Microsoft Defender for Cloud](https://aka.ms/mdc) to connect DevOps entities with their related resources running multi-cloud (Azure, AWS, GCP, on-premise).
```

Comment:
Grammar: "containers digest ID" should be either "container digest IDs" or "a container's digest ID"